backend/btrixcloud/crawlconfigs.py

```diff
@@ -1728,6 +1732,7 @@ async def get_crawl_configs(
                 description='Defaults to `"and"` if omitted',
             ),
         ] = ListFilterType.AND,
+        dedupeCollId: Optional[UUID] = None,
```

Comment:
```suggestion
        dedupe_coll_id: Annotated[
            Optional[UUID], Query(alias="dedupeCollId", title="Deduplication Source Collection")
        ] = None,
```
Just to keep consistent with the rest of this endpoint's args